dashboard/src/features/shared/components/SearchBar/SearchBar.scss

```diff
@@ -80,5 +79,6 @@
 .queryTime {
   display: inline-block;
   padding: 4px 20px;
-  opacity: 0.66
+  opacity: 0.66;
+  float: right;
```

Comment:
Is floating right a vestige of the previous design when there was a button on the left? If so, can we revert this? If not, let's use `text-align` instead of `float`. We don't use `float` anywhere else in our CSS.